community_images/consul/bitnami/dc_acl_coverage.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -x
+set -e
+
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# shellcheck disable=SC1091
+. "${SCRIPTPATH}"/../../common/scripts/bash_helper.sh
+
+JSON_PARAMS="$1"
+
+JSON=$(cat "$JSON_PARAMS")
+
+echo "Json params for docker compose coverage = $JSON"
+
+# Consul ACLs
+docker exec -i consul-server1 consul acl bootstrap
```

Comment:
may be add few more simple acl specific commands like adding/listing tokens

Reply:
Noted, Adding them right now.

Reply:
Update: The commands like adding acl tokens or listing them would take time. These commands require acl:write rule to be active. For this, We'll have to configure the acl.hcl files, dns/hcl files, give tokens, etc (total restructuring of docker-compose-acl.yml).
Ref: https://github.com/dj-wasabi/consul-acl-examples/tree/master/single-dc

I can do that if necessary. It'll take some time. Please give me a signal here.

Reply:
okay, i think it's fine for now to not add them since we are at least adding bootstrap for ACL